test/test_jit_disabled.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import sys
+import os
+import contextlib
+import subprocess
+
+
+@contextlib.contextmanager
+def _jit_disabled():
+    cur_env = os.environ.get("PYTORCH_JIT", "1")
+    os.environ["PYTORCH_JIT"] = "0"
+    try:
+        yield
+    finally:
+        os.environ["PYTORCH_JIT"] = cur_env
+
+_program_string = """
+import torch
+class Foo(torch.jit.ScriptModule):
+    def __init__(self, x):
+        super(Foo, self).__init__()
+        self.x = torch.jit.Attribute(x, torch.Tensor)
+
+    def forward(self, input):
+        return input
+
+s = Foo(torch.ones(2, 3))
+print(s.x)
+"""
+
+class TestJitDisabled(unittest.TestCase):
```

Comment:
Can you move https://github.com/pytorch/pytorch/blob/489cc46686e7e689e7130e78b73fe77a9edac5ed/test/test_jit.py#L2849 over here as well